content/contents+pt.lr

```diff
@@ -2,121 +2,121 @@ _model: home
 ---
 _template: home.html
 ---
-title: As IDEs do Python
+title: A IDE do Python
```

Comment:
`As IDEs` aqui seria a tradução mais correta aqui (tradução literal).
O trocadilho (com `The Ides of March`) infelizmente é intraduzível.